Assets/Scripts/Augment/AugmentImplementations/Solar/SolarBody.cs

```diff
@@ -24,6 +31,8 @@ public override void Start()
     {
         meshRenderer.materials[solarPanelMaterialIndex] = Instantiate(meshRenderer.materials[solarPanelMaterialIndex]);
         solarPanelMaterial = meshRenderer.materials[solarPanelMaterialIndex];
+        // The main directional light in the scene (the sun) should ideally be tagged as such, but this works for now
+        globalLightDirection = FindFirstObjectByType<Light>().transform.rotation;
```

Comment:
Could you specify `DirectionalLight` instead of just `Light`?

Reply:
The directional light class is unintuively not the Light we're looking for (Its part of an experimental global lighting package), and the light we want is a Light object instance with a directional property.